resources/views/components/order.blade.php

```diff
@@ -0,0 +1,32 @@
+@props(['order' , 'product'])
+
+<div class="rounded-md w-4/5 bg-gray-200 lg:h-[20vh] h-[75vh] flex lg:flex-row flex-col justify-center items-center p-[3%] lg:p-[1%]">
+    <div class="lg:w-1/6 w-5/6 h-2/3 lg:h-full flex justify-center items-center">
+        <img src="{{ asset($product->image_path) }}" alt="{{$product->name}}" class="object-contain h-full w-full">       
+    </div>
+    <div class="lg:w-5/6 w-5/6 h-1/3 lg:h-full mt-4 lg:mt-0 flex lg:flex-row flex-col items-center justify-between space-y-2">
+        <div class="flex flex-col h-2/3 lg:h-auto items-center lg:items-left lg:pl-[2%] lg:pr-[10%]">
+            <p class="text-5xl lg:text-xl h-1/2 lg:h-auto">
+                <time> {{substr($order->order_date, 0, 11)}} </time> | {{$order->id}}
+            </p>
+            @if($order->orderStatus->id == 1)
+            <p class="text-5xl lg:text-xl  text-red-500 h-1/2 lg:h-auto">
+                {{$order->orderStatus->status}}
+            </p>
+            @else
+                <p class="text-5xl lg:text-xl text-emerald-500 h-1/2 lg:h-auto">
+                    {{$order->orderStatus->status}}
+                </p>
+            @endif        
```

Comment:
Waarom bepaal je de kleur in de view? Mij lijkt het dat je dat ergens anders bepaalt.

Reply:
waar anders?